source/release/4.5.0/index.md

```diff
@@ -72,69 +76,72 @@ dnf config-manager --set-enabled ovirt-45-upstream-testing
 #### oVirt Engine
 
  - [BZ 2056126](https://bugzilla.redhat.com/show_bug.cgi?id=2056126) **[RFE] Extend time to warn of upcoming certificate expiration**
-   RHV Manager 4.4 SP1 certificate expiration check will warn of upcoming certificate expiration earlier:
+   oVirt Engine 4.5 certificate expiration check will warn of upcoming certificate expiration earlier:
 
 1. If a certificate is going to expire in the upcoming 120 days, then WARNING event is raised in the audit log
 2. If a certificate is going to expired in the upcoming 30 days, then ALERT event is raised in the audit log
 
-This is check is covering internal RHV certificates (for example certificate for RHVM &lt;-&gt; hypervisor communication), but it doesn't check for custom certificates configured for HTTPS access to RHVM as configured according to https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/administration_guide/appe-red_hat_enterprise_virtualization_and_ssl#Replacing_the_Manager_CA_Certificate
+This is check is covering internal oVirt certificates (for example certificate for oVirt Engine &lt;-&gt; hypervisor communication), but it doesn't check for custom certificates configured for HTTPS access to oVirt Egnine as configured according to https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/administration_guide/appe-red_hat_enterprise_virtualization_and_ssl#Replacing_the_Manager_CA_Certificate
  - [BZ 2021545](https://bugzilla.redhat.com/show_bug.cgi?id=2021545) **[RFE] Add cluster-level 4.7**
    In this release we have added DataCenter/Cluster compatibility level 4.7, which is available only on hosts with RHEL 8.6+, latest CentOS Stream 8 and CentOS Stream/RHEL 9 with libvirt &gt;= 8.0.0 installed.
  - [BZ 2015093](https://bugzilla.redhat.com/show_bug.cgi?id=2015093) **[RFE] Implement hypervirors core dump related configuration to work on EL8 and EL9**
    oVirt 4.5 has moved from abrt to systemd-coredump to gather crash dumps on the hypervisors
  - [BZ 2023250](https://bugzilla.redhat.com/show_bug.cgi?id=2023250) **[RFE] Use virt:rhel module instead of virt:av in RHEL 8.6+ to get advanced virtualization packages**
    Advanced Virtualization module (virt:av) has been merged into standard RHEL virtualization module (virt:rhel) as a part of RHEL 8.6 release. Due to this change the host deploy and host upgrade flows has been updated to properly enable virt:rhel module during new installation of RHEL 8.6 host and during upgrade of existing RHEL &lt;= 8.5 to RHEL 8.6 host.
- - [BZ 1940824](https://bugzilla.redhat.com/show_bug.cgi?id=1940824) **[RFE] Upgrade OVN/OVS 2.11 in RHV to OVN/OVS 2.15**
+ - [BZ 1940824](https://bugzilla.redhat.com/show_bug.cgi?id=1940824) **[RFE] Upgrade OVN/OVS 2.11 in oVirt to OVN/OVS 2.15**
    Upgrade from OvS/OVN 2.11 to OVN 2021 and OvS 2.15.
 
 The upgrade is transparent to user as long as few conditions are met:
 
 1) The engine must be upgraded first.
```

Comment:
Global change: This list is rendering as a paragraph. You should mark each line with "-" or "1." so that it appears as a list.

Reply:
Updated doctext at https://bugzilla.redhat.com/show_bug.cgi?id=1940824 will be fixed in next regeneration